sql/05-load-table-friend.sql

```diff
@@ -0,0 +1,21 @@
+INSERT INTO student.friends(id1, id2) VALUES
```

Comment:
i think it just assumes your user name if you don't create a schema. i think schema's have to deal with permission issues.